src/ROUTES.ts

```diff
@@ -117,6 +118,7 @@ export default {
     getMoneyRequestMerchantRoute: (iouType: string, reportID = '') => `${iouType}/new/merchant/${reportID}`,
     getMoneyRequestDistanceTabRoute: (iouType: string, reportID = '') => `${iouType}/new/${reportID}/distance`,
     getMoneyRequestWaypointRoute: (iouType: string, waypointIndex: number) => `${iouType}/new/waypoint/${waypointIndex}`,
+    getMoneyRequestTagRoute: (iouType: string, reportID = '', tag = '') => `${iouType}/new/tag/${reportID}/${tag}`,
```

Comment:
Hmm I don't think we want the tag list name in the route params unless absolutely necessary. Is this the only option?

Reply:
Right now we can hardcode to the first tag in policyTags in `MoneyRequestTagPage`. After that, we need to find a way of passing to the page the tag that is currently being altered.